### ISSUE TITLE
Need to enable LESS SECURE APP from user account setting

### ISSUE THREAD

**Issue by apmeena** [mscdex/node-imap#711]
This is a good module but we need to enable ALLOW LESS SECURE APP setting from our Gmail and Yahoo account to get access of inbox. Is there any way to avoid this setting?

Thanks.

**Comment by mscdex** [comment]
Yes, use OAuth to log in. See the `xoauth2` config setting in the readme.
